src/dso_api/lib/schematools/models.py

```diff
@@ -103,10 +122,18 @@ def fetch_crs(dataset: DatasetSchema) -> Dict[str, Any]:
         UnlimitedCharField
     ),
     "https://geojson.org/schema/Geometry.json": field_model_factory(
-        models.MultiPolygonField, value_getter=fetch_crs, srid=28992, geography=False
+        models.MultiPolygonField,
+        value_getter=fetch_crs,
+        srid=28992,
```

Comment:
Is de default altijd RD-NEW?

Reply:
Nice! Kunnen we nu idd gelijk gebruiken :-)